src/loki/search.cc

```diff
@@ -387,7 +374,8 @@ struct bin_handler_t {
         length_ratio = 1.f - length_ratio;
       }
       // side of street
-      auto side = candidate.get_side(location.latlng_, candidate.sq_distance);
+      auto sq_tolerance = location.street_side_tolerance_ * location.street_side_tolerance_;
+      auto side = candidate.get_side(location.latlng_, candidate.sq_distance, sq_tolerance);
```

Comment:
seemed odd to have the request parameter and the config be sq tolerance so we just square them as we go